graylog2-server/src/test/java/org/graylog2/plugin/MessageTest.java

```diff
@@ -638,4 +639,18 @@ public void assignUnsupportedTemporalTypeAsTimestamp() {
         message.addField(Message.FIELD_TIMESTAMP, ThaiBuddhistDate.of(0, 4, 19));
         assertThat(message.getTimestamp()).isGreaterThan(new DateTime(2018, 4, 19, 0, 0, 0, 0, DateTimeZone.UTC));
     }
+
+    @Test
+    public void testSetMessageState() throws NoSuchFieldException, IllegalAccessException {
+        final Message message = new Message("message", "source", Tools.nowUTC());
+        final Field stateMap = Message.class.getDeclaredField("metadata");
```

Comment:
This breaks in Jenkins and my local environment. I think it's okay to not test the internals here.